# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/search/tests/xpcshell/head_search.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c toolkit/components/search/tests/xpcshell/head_search.js
--- a/toolkit/components/search/tests/xpcshell/head_search.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/toolkit/components/search/tests/xpcshell/head_search.js	Tue Feb 12 16:08:25 2019 +0000
@@ -222,7 +222,7 @@
     uri: "resource://search-plugins/list.json",
     loadUsingSystemPrincipal: true,
   });
-  let searchSettings = parseJsonFromStream(chan.open2());
+  let searchSettings = parseJsonFromStream(chan.open());
   let defaultEngineName = searchSettings.default.searchDefault;
 
   if (isUS === undefined)
@@ -241,7 +241,7 @@
     uri: "resource://search-plugins/list.json",
     loadUsingSystemPrincipal: true,
   });
-  let json = parseJsonFromStream(chan.open2());
+  let json = parseJsonFromStream(chan.open());
   let visibleDefaultEngines = json.default.visibleDefaultEngines;
 
   if (isUS === undefined)